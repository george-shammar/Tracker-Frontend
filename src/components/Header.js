import React from 'react';
import '../stylesheets/header.css';

const Header = () => {
    return (
        <div className="nav">
          <div className="health-nav">
            <span className="material-icons logo-nav">maps_ugc</span>
            <p>Health<span className="p-subtitle">Track</span></p>
          </div>
        </div>
    );
  }
  
  export default Header;