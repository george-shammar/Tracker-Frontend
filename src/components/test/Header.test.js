import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../Header';

it('renders the Home component correctly', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});