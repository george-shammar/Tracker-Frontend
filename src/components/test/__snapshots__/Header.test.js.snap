// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Home component correctly 1`] = `
<div
  className="nav"
>
  <div
    className="health-nav"
  >
    <span
      className="material-icons logo-nav"
    >
      maps_ugc
    </span>
    <p>
      Health
      <span
        className="p-subtitle"
      >
        Track
      </span>
    </p>
  </div>
</div>
`;
